include/bluetooth/gap.h

```diff
@@ -189,6 +190,140 @@ enum {
 	BT_GAP_SCA_0_20 = 7,
 };
 
+/**
+ * @brief Encode 40 least significant bits of 64-bit LE Supported Features into array values
+ *        in little-endian format.
+ *
+ * Helper macro to encode 40 least significant bits of 64-bit LE Supported Features value into
+ * advertising data. The number of bits that are encoded is a number of LE Supported Features
+ * defined by BT 5.3 Core specification.
+ *
+ * Example of how to encode the `0x000000DFF00DF00D` into advertising data.
+ *
+ * @code
+ * BT_DATA_BYTES(BT_DATA_LE_SUPPORTED_FEATURES, BT_LE_SUPP_FEAT_40_ENCODE(0x000000DFF00DF00D))
+ * @endcode
+ *
+ * @param w64 LE Supported Features value (64-bits)
+ *
+ * @return The comma separated values for LE Supported Features value that
+ *         may be used directly as an argument for @ref BT_DATA_BYTES.
+ */
+#define BT_LE_SUPP_FEAT_40_ENCODE(w64) \
+	(((w64) >> 0) & 0xFF),  \
+	(((w64) >> 8) & 0xFF),  \
+	(((w64) >> 16) & 0xFF), \
+	(((w64) >> 24) & 0xFF), \
+	(((w64) >> 32) & 0xFF), \
+	(((w64) >> 40) & 0xFF), \
+	(((w64) >> 48) & 0xFF), \
+	(((w64) >> 56) & 0xFF)
```

Comment:
```suggestion
	(((w64) >> 32) & 0xFF)
```

Reply:
Ah... yes I've forgotten to remove it, thanks!